### ISSUE TITLE
Davos split plots

### ISSUE THREAD

**Issue by cbrownGTREE** [G-TREE/DataSynthesisWorkshop#9]
@scelmendorf @atrant 

Looking at and thinking about the Davos data, I think the best plan is to remove all caged subplots and only analyze uncaged data. So, to link to the provenance issue, Davos plots to subset and include will be uncaged (herb.treat=non.ex) and low provenances (seed.provenance.um.zone=kublis AND wiesen).

The only remaining issue there is that the Davos subplots are smaller than the standard GTREE plot used at other sites, so they will need to be scaled to match the 50 x 50 cm size of the rest of the data. The Davos subplots are 22.5 x 28 cm.

Yay? Nay?

**Comment by scelmendorf** [comment]
@cbrownGTREE I agree with removing all the caged subplots, and whatever you think is right for the provenance (I know less about the provenances of the other sites).

I don't know what you mean by scale to match the size of the rest of the plots.  I'm not sure how we would achieve that, I think this would show up as some sort of crowding effect, but given other plots already have variable numbers of seeds added (or *germination rates = differing effective numbers of seeds added) I think our densities are already all over the map.

**Comment by cbrownGTREE** [comment]
@scelmendorf Good point - hadn't thought about it that way. 

Are those all of the current issues dealt with now, then?
